atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/DbkvsPostgresTestSuite.java

```diff
@@ -86,7 +87,9 @@ public static DbKeyValueServiceConfig getKvsConfig() {
 
         return ImmutableDbKeyValueServiceConfig.builder()
                 .connection(connectionConfig)
-                .ddl(ImmutablePostgresDdlConfig.builder().build())
+                .ddl(ImmutablePostgresDdlConfig.builder()
+                        .compactInterval(HumanReadableDuration.milliseconds(10))
```

Comment:
Why does this use some custom class instead of a normal TimeInterval?

Reply:
I think its preferred and make the config time easier to reason about by including the unit in the value of the config parameter.